web_src/css/base.css

```diff
@@ -2096,3 +2095,10 @@ table th[data-sortt-desc] .svg {
   align-items: center;
   gap: .25rem;
 }
+
+/* to override fomantic's default display: block for ".menu .item", and use a slightly larger gap for menu item content */
+.ui.dropdown .menu.flex-items-block > .item {
+  display: flex !important;
+  align-items: center;
+  gap: .5rem;
+}
```

Comment:
Can this `flex-items-block` not appear alone? I find the name slightly confusing. Maybe add `menu` in the name somewhere to make it clear it only appliesto menus.

Reply:
Done, renamed it to `flex-items-menu`. The `flex-items` is intentionally chosen to match the `flex-items-*` family.